apps/etterlatte-saksbehandling-ui/package.json

```diff
@@ -8,5 +8,6 @@
     "husky": "^8.0.1",
     "lint-staged": "^14.0.1",
     "prettier": "^3.0.3"
-  }
+  },
+  "pre-commit": "lint-staged"
```

Comment:
Kjøres denne automatisk

Reply:
Jeg trodde den gjorde det allerede :P

Reply:
I pre-commit fila kjøres `cd ./apps/etterlatte-saksbehandling-ui && yarn lint-staged` kan det fjernes da?

Reply:
Siden vi først kjører ktlint i `pre-commit` kan det vel være greit å kjøre lint-staged der og?

Reply:
Ja, det virker som vi ikke trenger denne i tillegg til husky-kjøringen

Reply:
Den ligger i parent `package.json` som burde holde.

Reply:
Fikk pokker ikke `yarn lint-staged` til å fungere i går. Den endte bare med å ikke godkjenne filer for commit, og dermed lage tomme commits som ikke ble lagt ved. Den korrigerte heller ikke feilene i filene. 

Etter litt mer testing i dag fant jeg til slutt ut hva som var feilen. Satt på gamle versjoner av samtlige pakker fra ui-rot sin package.json. Kjørte en fersk install og vipps fungerte det uten `pre-commit` i package.json... 

Mistenker at jeg ikke er alene om å ha gammel moro her, så må få oppdatert pre-commit slik at vi alltid er sikre på at folk kjører samme versjon som ligger i package.json, ikke bare den de installerte ved oppsett av utviklermiljøet første gang.